dev/preview/workflow/preview/build.sh

```diff
@@ -15,8 +15,57 @@ ensure_gcloud_auth
 
 VERSION="$(preview-name-from-branch)-dev"
 
+# We have quite a few Leeway packages whose hash includes files from .gitignore.
+# Some of these files (such as build folders) are populated as part of our prebuilds
+# which means that the hash for these components will be different from a workspace
+# then in CI.
+#
+# We should fix the package definitions so that they don't include these files in the hash.
+#
+# For now we're simply cleaning the repository by deleting all files that are in .gitignore.
+# The files were produced using "git clean -ndfX"
+components_dir="$SCRIPT_PATH/../../../../components"
+install_dir="$SCRIPT_PATH/../../../../install"
+rm -rf \
```

Comment:
😢

Reply:
It's an ephemeral workspace, what's the worst that could happen 😉  Kidding aside, I'm only okay with this because we're going to remove it soon when we fix the real issue in https://github.com/gitpod-io/gitpod/issues/14157 ☺️